share/sysbuild/image_configurations/BOOTLOADER_image_default.cmake

```diff
@@ -27,3 +27,5 @@ foreach(loopkeytype ${keytypes})
     set_config_bool(${ZCMAKE_APPLICATION} ${loopkeytype} n)
   endif()
 endforeach()
+
+include(image_configurations/ALL_image_default.cmake)
```

Comment:
No need to place this line here.

The `IMAGE_CONF_SCRIPT` property supports lists exactly for the purpose of allowing multiple conf scripts.
https://github.com/zephyrproject-rtos/zephyr/blob/f34f06058890e8a2628429bf67917dc8d6996487/share/sysbuild/cmake/modules/sysbuild_extensions.cmake#L456-L458
https://github.com/zephyrproject-rtos/zephyr/blob/f34f06058890e8a2628429bf67917dc8d6996487/share/sysbuild/cmake/modules/sysbuild_extensions.cmake#L472

So just add `ALL_image_default.cmake` to that property for all images, regardless of boot / main images.

```suggestion
```

Reply:
Fixed